packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/remote-table.service.ts

```diff
@@ -181,14 +175,14 @@ export class RemoteTableService {
       localTableName,
       input,
       remoteServer,
-      remoteTableColumns,
+      distantTableColumns,
     );
 
     await this.createRemoteTableMetadata(
```

Comment:
This should be in a transaction to avoid having a foreign table created without metadata

Reply:
this requires a refacto that should not be done in this PR IMO. The service fieldMetadataService is doing a transaction for each creation. We would need to send a manager as input so it uses it when defined